packages/syft/src/syft/core/tensor/autograd/tensor.py

```diff
@@ -64,58 +85,62 @@ def __abs__(self) -> AutogradTensorAncestor:
         op = autograd.backward_ops.AbsOp()
         return op(self)
 
-    def __add__(self, other) -> AutogradTensorAncestor:
+    def __add__(self, other: AutogradTensor) -> AutogradTensorAncestor:
         op = autograd.backward_ops.AddOp()
         return op(self, other)
 
-    def __sub__(self, other) -> AutogradTensorAncestor:
+    def __sub__(self, other: AutogradTensor) -> AutogradTensorAncestor:
         op = autograd.backward_ops.SubOp()
         return op(self, other)
 
-    def __mul__(self, other) -> AutogradTensorAncestor:
+    def __mul__(self, other: AutogradTensor) -> AutogradTensorAncestor:
         op = autograd.backward_ops.MulOp()
         return op(self, other)
 
-    def __rmul__(self, other) -> AutogradTensorAncestor:
+    def __rmul__(self, other: AutogradTensor) -> AutogradTensorAncestor:
         op = autograd.backward_ops.MulOp()
         return op(self, other)
 
-    def __truediv__(self, other) -> AutogradTensorAncestor:
+    def __truediv__(self, other: AutogradTensor) -> AutogradTensorAncestor:
         if is_acceptable_simple_type(other):
             return self * (1 / other)
         return NotImplemented
 
-    def __pow__(self, other) -> AutogradTensorAncestor:
+    def __pow__(
+        self, other: Union[AutogradTensor, Union[int, bool, float, Any]]
```

Comment:
Union[AutogradTensor, Union[int, bool, float, Any]] is equivalent with Any I believe. Do we have any chance to narrow that Any?

Reply:
Great catch, I missed that. I made a commit to fix it. Thanks Tudor!